certora/specs/AllowancesInvariant.spec

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+using GeneralAdapter1 as GeneralAdapter1;
+using EthereumGeneralAdapter1 as EthereumGeneralAdapter1;
+using ParaswapAdapter as ParaswapAdapter;
+
+methods {
+    function _.approve(address token, address spender, uint256 amount)  external => summaryApprove(calledContract, spender, amount) expect bool;
+    // Aave dispatch
+    function _.repay(address, uint256, uint256, address) external => HAVOC_ECF;
```

Comment:
Why is this summary useful ? The default summary here would be HAVOC_ECF I think